lib/screens/item/widgets/main_view.dart

```diff
@@ -484,18 +484,40 @@ class _ParentItemSection extends StatelessWidget {
                   ],
                   const Spacer(),
                   if (!state.isOfflineReading)
-                    CustomDropdownMenu<FetchMode>(
-                      menuChildren: FetchMode.values,
-                      onSelected: context.read<CommentsCubit>().updateFetchMode,
-                      selected: state.fetchMode,
+                    Semantics(
+                      container: true,
+                      label: 'Fetch Mode',
+                      hint: 'Select fetch mode for comments',
+                      child: GestureDetector(
+                        onTap: () {},
```

Comment:
Seems like there are two spaces here
```suggestion
                        onTap: () {},
```